octarine/install-octarine.go

```diff
@@ -0,0 +1,113 @@
+package octarine
+
+import (
+	"io/ioutil"
+	"math/rand"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+const (
+	nsMgrUsername      = "meshery"
+	nsMgrPassword      = "meshery-password"
+	octarineDeployment = "meshery:playground"
```

Comment:
@thehh1974 quick check - are we good with having these credentials public?

Reply:
Better to have it as run time params or environment variables which are injected dynamically.